app/assets/stylesheets/custom.css.scss

```diff
@@ -99,19 +108,30 @@ p {
 	text-decoration: none;
 }
 
-
 footer {
-	position: fixed;
-	bottom: 0;
-	width: 100%;
-	padding: 1rem;
-	text-align: center;
+	margin-top: 45px;
```

Comment:
Again, this is the previous version of css before I pushed, but that's fine, I will just merge again after we merge this

Reply:
`git pull origin main` @mrubakh

Reply:
I totally did that... git is so annoying